modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,288 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-cluster-using-defaults-ocm_{context}"]
+= Creating a cluster with the default options using {cluster-manager}
+
+When using {cluster-manager-first} to create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can select the default options to create the cluster quickly.
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
++
+[NOTE]
+====
+To successfully install ROSA 4.10 clusters, use the latest version of the ROSA CLI.
+====
+* You have logged in to your Red Hat account by using the `rosa` CLI.
+* You have verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select *Create cluster*.
+
+. On the *Create an OpenShift cluster* page, select *Create cluster* in the *{product-title}* row.
+
+. Review and complete the *Prerequisites* listed on the *Accounts and roles* page. Select the checkbox to acknowledge that you have read and completed all of the prerequisites.
+
+. Select an AWS account from the *Associated AWS account* drop-down menu. If no associated AWS accounts are found, click *Associate AWS account* and follow these steps:
+.. On the *Authenticate* page, click the copy button next to the `rosa login` command. The provided command includes your ROSA API login token.
```

Comment:
somewhere here or near/after the command that uses the login token, can we include a tip to tell the user that if they need a new rosa api login token, they can also use this: https://console.redhat.com/openshift/token  ?

Reply:
@arendej thanks for your review and for this suggestion. I have added a note under this step in this procedure and the using OCM with customizations procedure.